js/deer-record.js

```diff
@@ -297,6 +298,7 @@ export default class DeerReport {
                         target: entity["@id"],
                         body: {}
                     }
+                    if(this.context) { annotation["@context"] = UTILS.processContextSyntax(this.context) } 
```

Comment:
If we tracked context per input that will be an annotation, here is where we would handle it.  Right now, it just takes the context the DeerRecord knows from the form, which created the original entity these annotations are targeting.